frontend/views/components/GroupSettings/InviteLinks.vue

```diff
@@ -109,72 +114,94 @@ export default {
-            isAnyoneLink: false,
-            state: {
-              description: 'Used',
-              expireInfo: '',
-              isExpired: true
-            }
-          },
-          {
-            name: 'Carl Sagan',
-            inviteLink: 'http://localhost:8000/app/join?groupId=B1gfSSeZc721XWnNFz7RbkoyHUqAeSwLT1cNHnnCssmSw6dJev&secret=1348',
-            isAnyoneLink: false,
-            state: {
-              description: 'Not used',
-              expireInfo: 'Expired',
-              isExpired: true
-            }
-          },
-          {
-            name: 'Anyone',
-            inviteLink: 'http://localhost:8000/app/join?groupId=s8LT1cNHnnCs21XWnNFz7RbNPKHUqAesmSw6dJeB1gfSSeZc7v&secret=5521',
-            isAnyoneLink: true,
-            state: {
-              description: '10/60 used',
-              expireInfo: 'Expired',
-              isExpired: true
-            }
-          }
-        ]
+        }
       }
     }
   },
   methods: {
     unfocusSelect () {
       this.$refs.select.blur()
     },
-    toInvite (e) {
-      if (e.target.tagName === 'SPAN') this.$router.push({ path: '/invite' })
-    },
     activateWebShare (inviteLink) {
       if (navigator.share) {
         navigator.share({
-          title: this.L('Your invite'),
+          title: L('Your invite'),
           url: inviteLink
         })
       }
+    },
+    inviteStatusDescription ({
+      isAnyoneLink,
+      isInviteExpired,
+      isAllInviteUsed,
+      quantity,
+      numberOfResponses
+    }) {
+      if (isAnyoneLink) return L('{numberOfResponses}/{quantity} used', { numberOfResponses, quantity })
+      else if (isAllInviteUsed) return L('Used')
+      else return isInviteExpired ? L('Not used') : L('Not used yet')
+    },
+    readableExpiryInfo (expiryTime) {
+      const MIL = 1000
+      const MIL_MIN = 60 * MIL
+      const MIL_HR = MIL_MIN * 60
+      const MIL_DAY = 24 * MIL_HR
+      let remainder
+
+      const days = Math.floor(expiryTime / MIL_DAY)
+      remainder = expiryTime % MIL_DAY
+      const hours = Math.floor(remainder / MIL_HR)
+      remainder = remainder % MIL_HR
+      const minutes = Math.ceil(remainder / MIL_MIN)
+      const options = [
+        undefined,
+        L('{minutes}m left', { minutes }),
+        L('{hours}h {minutes}m left', { hours, minutes }),
+        L('{days}d {hours}h {minutes}m left', { days, hours, minutes })
+      ]
+
+      return options[!!days + !!hours + !!minutes]
+    },
+    mapInvite ({
+      creator,
+      inviteSecret,
+      responses,
+      quantity,
+      expires: expiryTime
+    }) {
+      const isAnyoneLink = creator === 'INVITE_INITIAL_CREATOR'
+      const isInviteExpired = expiryTime <= 0
+      const numberOfResponses = Object.keys(responses).length
+      const isAllInviteUsed = numberOfResponses === quantity
+
+      return {
+        isAnyoneLink,
+        invitee: isAnyoneLink ? 'Anyone' : Object.keys(responses)[0],
```

Comment:
@sandrina-p  `Object.keys(responses)[0]` here is because I couldn't find `invitee` property from the raw invite data fetched from the store.

Reply:
Oh, it's a bug! Checked and it's a quick fix. `ProposalVoteOptions.vue`, when calling `createInvite()` at line ~105.

```diff
- payload.passPayload = createInvite({ creator: this.proposal.data.proposalData.member })
+ payload.passPayload = createInvite({ invitee: this.proposal.data.proposalData.member, creator: this.proposal.meta.username })
```

Reply:
Btw, it's missing L around Anyone.

Reply:
Thanks! It's fixed that way.